src/components/types.ts

```diff
@@ -4,18 +4,23 @@ import type { EmbedOptions, VisualizationSpec } from 'vega-embed'
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type MessageData = { [key: string]: any }
 
+export interface Sender {
+  id: string
+  name?: string
+}
+
 export interface Message {
   id: string
   timestamp: string
-  sender: string
+  sender: Sender
   conversationId: string
   format: string
   data: MessageData
   inReplyTo?: string
   threadId?: string
   priority?: string
   taggedParticipants?: string[]
-  topicId?: string
+  topic?: string
```

Comment:
Did you change this unintentionally?

Reply:
Some components used `topicId` in the storybook and cypress. Could you search `topicId` and rename it? MessageSpace's description for `messages` also contains `topicId`